src/pool/srv_util.c

```diff
@@ -306,14 +306,14 @@ ds_pool_map_tgts_update(struct pool_map *map, struct pool_target_id_list *tgts,
 				target->ta_comp.co_rank,
 				target->ta_comp.co_index, map);
 			target->ta_comp.co_status = PO_COMP_ST_DOWN;
-			target->ta_comp.co_fseq = version;
+			target->ta_comp.co_fseq = version++;
```

Comment:
just to confirm that if one server (with 8 vos tgts) failed and exclude the server, then each vos target with different co_fseq, and next pool map ver will +8?  is it to fix original code's what issue?

Reply:
oh, yes, that is this patch intends to do, because the current spare target selection algorithm requires each target has to have different failure sequence.

Reply:
I think we eventually want them to have the same failure sequence to parallelize rebuild right?

Reply:
sorry I misunderstood this